src/test/java/com/github/dariobalinzo/filter/JsonCastFilterTest.java

```diff
@@ -32,17 +32,19 @@
 import static junit.framework.TestCase.assertEquals;
 
 public class JsonCastFilterTest {
+    private static final boolean IS_WINDOWS = System.getProperty( "os.name" ).contains( "indow" );
```

Comment:
is this really needed? what was your issue with windows? the file is not in the file system, but inside the classpath, so i expect this to work also there.